docs/useSessionstorageState.md

```diff
@@ -4,6 +4,12 @@ title: useSessionstorageState
 sidebar_label: useSessionstorageState
 ---
 
+:::warning
+
+`useSessionstorageState` is deprecated, it will be removed in rooks v7. Please use [useSessionstorage](/useSessionstorage) instead.
```

Comment:
I think I didn't explain this well. We should be deprecate `useSessionstorage`. It's buggy. Apologies. 😓